Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -892,7 +900,7 @@ extension BrowserViewController: HomePanelViewControllerDelegate {
         finishEditingAndSubmit(url, visitType: visitType)
     }
     func homePanelViewController(homePanelViewController: HomePanelViewController, didSelectPanel panel: Int) {
-        tabManager.selectedTab?.lastHomePanel = panel
+        tabManager.selectedTab?.webView?.evaluateJavaScript("history.replaceState({}, '', '/about/home/#panel=\(panel)')", completionHandler: nil)
```

Comment:
i did this but idk what this does, explain it to me when you get back from your disappearance